dl/dl-python/dl-python.jinja

```diff
@@ -127,7 +127,7 @@ RUN pip --no-cache-dir install \
         sympy \
         Pillow \
         zmq \
-        opencv-contrib-python==3.4.0.12 \
+        opencv-contrib-python \
```

Comment:
I think we forgot to update it manually to the same version of OPENCV.

Reply:
Should we keep this in sync with the OPENCV version in dl-deps? If so, we need to pass a param with the same version. Right?

Reply:
Actually I think we should remove this since it isn't the official repo but unofficial pre-built OpenCV packages for Python. Given that we are installing OPENCV from source we should definitively remove this.